docs/index.rst

```diff
@@ -449,6 +449,78 @@ following flag or environment variable::
+
+The Datadog opentracer provides an OpenTracing-compatible API to the Datadog
+tracer so that you can use the Datadog tracer in your OpenTracing-compatible
+applications.
+
+Installation
+~~~~~~~~~~~~
+
+To include OpenTracing dependencies in your project with `ddtrace`, ensure you
+have the following in `setup.py`::
+
+    install_requires=[
+        "ddtrace[opentracing]",
+    ],
+
+Configuration
+~~~~~~~~~~~~~
+
+The OpenTracing convention for initializing a tracer is to define an
+initiliazation method that will configure and instantiate a new tracer and
+overwrite the global `opentracing.tracer` reference.
+
+Typically this method looks something like::
+
+    from opentracing.scope_managers import ThreadLocalScopeManager
+    from ddtracer.opentracer import Tracer
+
+    def init_tracer(service_name):
+        config = {
+          'agent_hostname': 'localhost',
+          'agent_port': 8126,
+          'debug': False,
+          'enabled': True,
+          'global_tags': {},
+        }
+        return Tracer(service_name, config=config, scope_manager=ThreadLocalScopeManager)
+
+
++---------------------+---------------------------------------------------------+---------------+
+| Configuration Key   |  Description                                            | Default Value |
++=====================+=========================================================+===============+
+| `enabled`           | enable or disable the tracer                            | `True`        |
++---------------------+---------------------------------------------------------+---------------+
+| `debug`             | enable debug logging                                    | `False`       |
++---------------------+---------------------------------------------------------+---------------+
+| `agent_hostname`    | hostname of the Datadog agent to use                    | `localhost`   |
++---------------------+---------------------------------------------------------+---------------+
+| `agent_port`        | port the Datadog agent is listening on                  | `8126`        |
++---------------------+---------------------------------------------------------+---------------+
+| `global_tags`       | tags that will be applied to each span                  | `{}`          |
++---------------------+---------------------------------------------------------+---------------+
+| `sampler`           | see `Sampling`_                                         | `AllSampler`  |
++---------------------+---------------------------------------------------------+---------------+
+| `priority_sampling` | see `Priority Sampling`_                                | `False`       |
++---------------------+---------------------------------------------------------+---------------+
+| `settings`          | see `Advanced Usage`_                                   | `{}`          |
++---------------------+---------------------------------------------------------+---------------+
+
+Usage
```

Comment:
Now that I think about it, would be great if you can put some examples directly here like we do for Java docs: https://github.com/DataDog/documentation/blob/bde053b60376d517961c46047d2be0a0254cf75c/content/tracing/setup/java.md#opentracing-api

You can use the same examples, translated in Python. When we'll refactor the docs, we'll move that section in the right place.